utils/parsing.go

```diff
@@ -267,7 +286,8 @@ func ParseSourceIdAndCohortIdAndVariablesList(c *gin.Context) (int, int, []int64
 	if err != nil {
 		return -1, -1, nil, nil, err
 	}
-	conceptIds, cohortPairs := GetConceptIdsAndCohortPairsAsSeparateLists(conceptIdsAndCohortPairs)
+	conceptIdsAndValues, cohortPairs := GetConceptIdsAndCohortPairsAsSeparateLists(conceptIdsAndCohortPairs)
+	conceptIds := ExtractConceptIdsFromCustomConceptVariablesDef(conceptIdsAndValues)
```

Comment:
maybe move `ExtractConceptIdsFromCustomConceptVariablesDef` into `GetConceptIdsAndCohortPairsAsSeparateLists` ?

Reply:
but you can still use it inside `GetConceptIdsAndCohortPairsAsSeparateLists`, right?

Reply:
`ExtractConceptIdsFromCustomConceptVariablesDef` is created to give a convenient way to plug things into existing functions which only support concept ids to limit the scope of this PR.

`GetConceptIdsAndCohortPairsAsSeparateLists` is used in `ParseConceptIdsAndDichotomousDefs` which in turn is used in the new stat endpoint, so we need them to pass back both ids and values. If we use 'ExtractConceptIdsFromCustomConceptVariablesDef inside `GetConceptIdsAndCohortPairsAsSeparateLists` then we will loss the value part.